hack/setup-scaffolding.sh

```diff
@@ -88,6 +98,7 @@ make ko-apply-tuf
 kubectl -n ctlog-system get secrets ctlog-public-key -oyaml | sed 's/namespace: .*/namespace: tuf-system/' | kubectl apply -f -
 kubectl -n fulcio-system get secrets fulcio-pub-key -oyaml | sed 's/namespace: .*/namespace: tuf-system/' | kubectl apply -f -
 kubectl -n rekor-system get secrets rekor-pub-key -oyaml | sed 's/namespace: .*/namespace: tuf-system/' | kubectl apply -f -
+kubectl -n tsa-system get secrets tsa-pub-key -oyaml | sed 's/namespace: .*/namespace: tuf-system/' | kubectl apply -f -
```

Comment:
also, what is pub-key in this case? like fulcio, we will need to put certificates into the TUF targets. you can fetch it from `api/v1/timestamp/certchain`, then you'll need to split it up into each certificate (the chain will contain a leaf, any number of intermediates, and a root)

Reply:
I'll do that in a follow up, cool? Right now it stands up and nothing in the tests (from the tsa README) requires it, so seems like standing this up and testing to make sure it works seems like a good first step?

Reply:
yep, sg!